hooks/useNWACObservations.ts

```diff
@@ -17,30 +17,34 @@ import {ZodError} from 'zod';
 export const useNWACObservations = (center_id: AvalancheCenterID, endDate: RequestedTime) => {
   const {nwacHost} = React.useContext<ClientProps>(ClientContext);
   const {logger} = React.useContext<LoggerProps>(LoggerContext);
-  const key = queryKey(nwacHost, center_id);
+  const key = queryKey(nwacHost, center_id, endDate);
   const thisLogger = logger.child({query: key});
   thisLogger.debug('initiating query');
-  const fetchNWACObservationsPage = async ({pageParam = formatRequestedTime(endDate)}): Promise<ObservationsQueryWithMeta> => {
-    const endDate: Date = requestedTimeToUTCDate(parseRequestedTimeString(pageParam));
-    const startDate = sub(endDate, {weeks: 2});
-    return fetchNWACObservations(nwacHost, center_id, startDate, endDate, thisLogger);
+  const fetchNWACObservationsPage = async (props: {pageParam?: unknown}): Promise<ObservationsQueryWithMeta> => {
```

Comment:
there was a bug here - the pageParam variable didn't get a default assignment because it wasn't undefined, it was null.

Reply:
Gotta love JavaScript :melting_face: